contract_api/registry.py

```diff
@@ -0,0 +1,273 @@
+import datetime
+import json
+from collections import defaultdict
+from common.utils import Utils
+from pymysql import MySQLError
+from contract_api.filter import Filter
+
+
+class Registry:
+    def __init__(self, obj_repo):
+        self.repo = obj_repo
+        self.obj_utils = Utils()
+
+    def _get_all_service(self):
+        """ Method to generate org_id and service mapping."""
+        try:
+            all_orgs_srvcs_raw = self.repo.execute("SELECT O.org_id, O.owner_address, S.service_id  FROM service S, "
+                                                   "organization O WHERE S.org_id = O.org_id AND S.is_curated = 1")
+            all_orgs_srvcs = {}
+            for rec in all_orgs_srvcs_raw:
+                if rec['org_id'] not in all_orgs_srvcs.keys():
+                    all_orgs_srvcs[rec['org_id']] = {'service_id': [],
+                                                     'owner_address': rec['owner_address']}
+                all_orgs_srvcs[rec['org_id']]['service_id'].append(
+                    rec['service_id'])
+            return all_orgs_srvcs
+        except Exception as e:
+            print(repr(e))
+            raise e
+
+    def _get_all_members(self):
+        """ Method to generate org_id and members mapping."""
+        try:
+            all_orgs_members_raw = self.repo.execute(
```

Comment:
This is a full table scan. Shouldnt it be bounded by orgID?

Reply:
/org API gives details of all organization.
Currently /org gives a list of all organization details. With an increase in no of organizations, we will be needing pagination support
Created Issue #155